dnf.spec

```diff
@@ -174,6 +174,9 @@ mv $RPM_BUILD_ROOT%{_bindir}/dnf-automatic-2 $RPM_BUILD_ROOT%{_bindir}/dnf-autom
 rm $RPM_BUILD_ROOT%{_bindir}/dnf-automatic-3
 %endif
 
+# This will eventually be the new default location for repo files
+mkdir $RPM_BUILD_ROOT%{_sysconfdir}/distro.repos.d
```

Comment:
@Conan-Kudo  would having just `%ghost /etc/dnf.repos.d` in the spec be sufficient for Mageia? We should not create that directory otherwise it would be used as default when others not exist. So f24 user could be confused.